docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -839,8 +839,43 @@
         format:</para>
 
         <programlisting>ds := DATASET('~remote::hpcc2::somescope1::somelfn1', rec);</programlisting>
+      </sect3>
+
+      <sect3>
+        <title>Preferred Storage</title>
+
+        <para>An option available for each type of cluster; hThor, Thor, and
+        Roxie is the <emphasis>preferredReadPlanes</emphasis> option. </para>
+
```

Comment:
<para>The <emphasis>preferredReadPlanes</emphasis> option is available for each type of cluster--hThor, Thor, and
        Roxie.</para>